### PR TITLE
syntactic sugar and made apiStub replaceable

### PR DESCRIPTION
**Pull request by olensmar**
see https://github.com/SmartBear/apispy-backend/blob/integration-tests/modules/integration-tests/src/test/java/com/smartbear/apispy/backend/ApiSpyBackendIT.java for how these are used

### CODE REVIEW

File: src/main/java/com/smartbear/readyapi/client/TestRecipeBuilder.java [hunk 1]
```diff
@@ -54,5 +54,14 @@ public static TestRecipeBuilder newTestRecipe() {
         return new TestRecipeBuilder();
     }
 
+    public static TestRecipeBuilder newTestRecipe(Object... args) {
+        TestRecipeBuilder recipeBuilder = newTestRecipe();
+        for (Object arg : args) {
```
Comment (veinSmart) [inline src/main/java/com/smartbear/readyapi/client/TestRecipeBuilder.java]: If you want to, you could change this to:
Arrays.stream(args).forEach(...);

File: src/main/java/com/smartbear/readyapi/client/support/AssertionUtils.java [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+import static org.junit.Assert.assertNotNull;
+
+public class AssertionUtils {
+
+    public static final Logger LOG = LoggerFactory.getLogger(AssertionUtils.class);
+
+    public static void assertExecution(Execution execution) {
+        assertNotNull(execution);
+
+        for (TestStepResult result : execution.getExecutionResult().getTestStepResults()) {
```
Comment (veinSmart) [inline src/main/java/com/smartbear/readyapi/client/support/AssertionUtils.java]: Same thing here, you could change this to:
... getTestStepResults().forEach(...) ...

File: src/main/java/com/smartbear/readyapi/client/TestRecipeBuilder.java (hunk 1 above)
Comment (olensmar) [inline src/main/java/com/smartbear/readyapi/client/TestRecipeBuilder.java]: don't get me wrong - I'd love to go 1.8 :-) perhaps if we add that "extractor" thing!?

**Comment by olensmar** [comment]
@prakashmrec @veinSmart ok to merge?

File: src/main/java/com/smartbear/readyapi/client/TestRecipeBuilder.java [hunk 3]
```diff
@@ -54,5 +54,14 @@ public static TestRecipeBuilder newTestRecipe() {
         return new TestRecipeBuilder();
     }
 
+    public static TestRecipeBuilder newTestRecipe(Object... args) {
```
Comment (prakashmrec) [inline src/main/java/com/smartbear/readyapi/client/TestRecipeBuilder.java]: We should add some javadoc about what these arguments are supposed to be?

**Comment by prakashmrec** [comment]
Looks good now, merging! Thanks.

**Pull request merged**